docs/contributing.md

```diff
@@ -49,17 +48,17 @@ After your local installation is set up, make sure you've enabled `debug` mode i
 Flarum's front-end code is written in ES6 and transpiled into JavaScript. During development you will need to recompile the JavaScript using [Node.js](https://nodejs.org/). **Please do not commit the resulting `dist` files when sending PRs**; this is automatically taken care of when changes are merged into the `master` branch.
 
 ```bash
-cd packages/core/js
+cd packages/framework/framework/core/js
 npm install
 npm run dev
 ```
 
 The process is the same for extensions.
 
 ```bash
-cd packages/tags/js
+cd packages/framework/extensions/tags/js
 npm install
-npm link ../../core/js
+npm link ../../../../framework/core/js
 npm run dev
 ```
```

Comment:
I missed this unnecessary bit. Yarn workspaces for the win, I should go through mentions of `npm` and change them to `yarn` as that's what we use now in the monorepo.

Reply:
I am very happy with this change :grin: